Vungle/src/main/java/com/mopub/mobileads/VungleExtrasBuilder.java

```diff
@@ -0,0 +1,71 @@
+package com.mopub.mobileads;
+
+import com.vungle.warren.AdConfig;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A helper class for creating a network extras map that can be passed to the adapter to make
+ * customizations specific to Vungle.
+ */
+public final class VungleExtrasBuilder {
+
+    private static final String EXTRA_START_MUTED = "startMuted";
```

Comment:
we already have the same in VungleMediationSettings

Reply:
@alexander-gubov Currently we are moving ad settings from app level to placement level, so some of them are duplicates. It's a good suggestion, May refactor these common settings in one super class in future. I backlog this issue.